### PR TITLE
feat: filter on favorite pets added

### PR DESCRIPTION
**Pull request by Waishnav**
**Changes proposed:**
- Dropdown section for filter on click.

image.
Added a right-side dropdown menu to maintain hierarchy among the dropdown menus across the website.

### CODE REVIEW

File: src/components/pets/Favorites.tsx [hunk 1]
```diff
@@ -70,34 +72,100 @@ export default function Favorites() {
...
+    let filtered;
+    if (petType === "All-favorites") {
+      filtered = petList;
+    } else {
+      filtered = petList?.filter((pet) => pet?.type === petType);
+    }
+    return filtered;
+  }
+
+  const handleClick = (e: any) => {
```
Comment (redxzeta) [inline src/components/pets/Favorites.tsx:85]: Can u fix the lint issue

**Review by redxzeta — CHANGES_REQUESTED** [review]
Lgtm just need to fix the any lint issue

File: src/components/pets/Favorites.tsx (hunk 1 above)
Comment (Waishnav) [reply src/components/pets/Favorites.tsx:85]: Both on neovim and Vscode there is no listing issue showing.

Comment (Waishnav) [reply src/components/pets/Favorites.tsx]: solved the lint issue

**Review by redxzeta — APPROVED** [review]

**Pull request merged**